lua/wire/client/cl_wire_map_interface.lua

```diff
@@ -1,89 +1,241 @@
--- The client part of the wire map interface.
--- It's for the clientside wire ports adding and removing, also for the rendering stuff.
--- It's in in this folder, because point entities are serverside only.
+-- Clientside functionalities of Wire Map Interface
+-- This is mostly for predection and rendering
 
--- Removing wire stuff and other changes that were done.
-local OverRiddenEnts = {}
-local function RemoveWire(Entity)
-	if (not IsValid(Entity)) then return end
+local WIRE_CLIENT_INSTALLED = WIRE_CLIENT_INSTALLED
```

Comment:
the check for `WIRE_CLIENT_INSTALLED` can probably removed as its pretty pointless.
`WIRE_CLIENT_INSTALLED` being defined though, it seems it only ever gets used in the map interface code, does get used sometimes in external addons though so probably not removable

Reply:
Yeah `WIRE_CLIENT_INSTALLED` is an Wiremod library thing for those addons check if Wiremod was installed on the client side. The check in the WMI is a leftover. I have removed it. Thanks for pointing that out.